db/seeds.rb

```diff
@@ -5,3 +5,12 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+
+User.create(id: 1, name: 'Jay', nickname: 'dadjokes')
```

Comment:
Generally, you're going to want to leave assigning IDs to the database. If it's bothering you that the `id` values don't start from 1 when you reseed, you can have your seed file destroy all records, and also find a gem that resets primary key values.